python/hail/tests/test_api.py

```diff
@@ -783,11 +783,8 @@ def test_filter_partitions(self):
     def test_from_rows_table(self):
         ds = hl.import_vcf(resource('sample.vcf'))
         rt = ds.rows()
-        rm = hl.MatrixTable.from_rows_table(rt)
-        # would be nice to compare rm to ds.drop_cols(), but the latter
-        # preserves the col, entry types
-        self.assertEqual(rm.count_cols(), 0)
-        self.assertTrue(rm.rows()._same(rt))
+        rm = hl.MatrixTable.from_rows_table(rt, partition_key='locus')
```

Comment:
suggest renaming `ds` => `mt` and `rm` => `rmt`

Reply:
This wasn't something I changed.  I just made the test better